common/src/main/java/org/dash/wallet/common/ui/inputs/InputWrapper.kt

```diff
@@ -74,4 +90,34 @@ class InputWrapper(context: Context, attrs: AttributeSet): TextInputLayout(conte
     override fun isErrorEnabled(): Boolean {
         return isErrorEnabled
     }
+
+    override fun setCounterEnabled(enabled: Boolean) {
+        super.setCounterEnabled(enabled)
+
+        if (enabled) {
+            val counter = findViewById<TextView>(com.google.android.material.R.id.textinput_counter)
+            val params = (counter.layoutParams as MarginLayoutParams)
+            params.updateMarginsRelative(
+                top = resources.getDimensionPixelOffset(R.dimen.char_counter_top_padding),
+                end = resources.getDimensionPixelOffset(R.dimen.char_counter_end_padding)
+            )
+        }
+    }
+
+    private fun setPadding() {
+        var topPadding = 0
+        var startPadding = 0
+        var endPadding = 0
+        var bottomPadding = 0
+
+        if (isHintEnabled) {
+            topPadding = resources.getDimensionPixelOffset(R.dimen.input_bottom_padding)
+        }
+
+        if (isCounterEnabled) {
+            bottomPadding = resources.getDimensionPixelOffset(R.dimen.char_counter_bottom_padding)
+        }
+
+        setPaddingRelative(startPadding, topPadding, endPadding, bottomPadding)
+    }
```

Comment:
This is from Private Memo UI story. Backported in case this input is needed for anything else.